### ISSUE TITLE
In Scala 2.11 InstrumentedHandlerSuite fails

### ISSUE THREAD

**Issue by dkowis** [rackerlabs/api-checker#266]
The requested MBean is not loaded.

This is requested: 
```
"com.rackspace.com.papi.components.checker.handler":type="InstrumentedHandler",scope="MyInstTestValidator",name="S0"
```

I've got a huge debug log of all the mBeans that are registered, not a one of them is S0: https://gist.github.com/dkowis/02e3f8c342b49743b263

This doesn't happen in 2.10. I wonder if it's got something to do with a lot of Deprecated code in 2.11. -- https://gist.github.com/dkowis/625b816164e83b9bbd81

**Comment by dkowis** [comment]
Adding an eventually around it to account for race conditions did not resolve the problem in 10 seconds.

**Comment by wdschei** [comment]
This is addressed by #297.
